level-1/평균-구하기.js

```diff
@@ -30,3 +30,12 @@ function solution(arr) {
   });
   return answer / arr.length;
 }
+
+//정답 5 - yongchanson
+function solution(arr) {
+  let sum = 0;
+  for (var i = 0; i < arr.length; i++) {
+    sum += arr[i];
+  }
```

Comment:
🌱 반영해도 좋고 넘어가도 좋아요. 🌱

reduce 메서드를 사용해보시는 것은 어떨까요?
```suggestion
  const sum = arr.reduce((acc,curr)=>acc+curr,0)
```

Reply:
경현님 추천대로 수정하면 채린님 풀이와 너무 겹쳐서
삭제하고 좋은 방법 생기면 추가하도록 하겠습니다.